rpcserver.go

```diff
@@ -5335,6 +5335,21 @@ func (r *rpcServer) GetNodeInfo(ctx context.Context,
 				return nil
 			}
 
+			// Since ForEachChannel will give us the outgoing policy
```

Comment:
What about the effect of https://github.com/lightningnetwork/lnd/pull/3427/files#diff-674c79ba02b6e9c45a994388392689814f3538800351b2fd43fb3ade21e1effbL3773 ?

Ideally we get the sorting right in _once place_ so everything else is corrected as well.